src/components/customized-components/HomePageContentCardPreview/HomePageContentCardPreview.component.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import cardHomePage1 from "../../../assets/cardImg1.png";
+import cardHomePage2 from "../../../assets/cardImg2.png";
+import cardHomePage3 from "../../../assets/cardImg3.png";
+
+import HomePageContentCardComponent from "../HomePageContentCard/HomePageContentCard.component";
+
+import "./HomePageContentCardPreview.style.css";
+
+const HomePageContentCardPreviewComponent = () => {
```

Comment:
HomPagePreviewContentCard

Reply:
it is a list of HomePageContentCard. Maybe I change it into HomePageContentCardList ?